rest_framework_inclusions/renderer.py

```diff
@@ -61,6 +60,10 @@ def _render_inclusions(self, data, renderer_context):
                     continue
                 render_data[key] = value
 
+        # map the meta information, if any
+        render_data["data"] = serializer_data
+        render_data["inclusions"] = inclusions
```

Comment:
`data` and `inclusions` these could also be configurable class attributes to make it a bit more configurable